acceptance/tests/options/color.rb

```diff
@@ -0,0 +1,15 @@
+# This test is intended to ensure with --debug and --color, facter sends escape sequences to colorize the output
+test_name "C864545: --debug and --color command-line options should print DEBUG messages with color escape sequences" do
+  tag 'risk:high'
+
+  confine :except, :platform => 'windows' # On windows we don't get an escape sequence to detect to color change
+
+  agents.each do |agent|
+    step "Agent #{agent}: retrieve debug info from stderr using --debug and --color option" do
+      on(agent, facter('--debug --color'), :environment => { 'TERM' => 'xterm-256color' }) do |facter_output|
```

Comment:
Hmm. Didn't try that. I was trying to give the CI the best chance of passing as the escape sequence would be different if it was a different terminal type (assuming it supports color)